common/Projectile.js

```diff
@@ -0,0 +1,35 @@
+import nengi from "nengi";
+
+class Projectile {
+    constructor(x, y, rotation) {
+        this.x = x
+        this.y = y
+        this.rotation = rotation
+        this.speed = 100
+        this.velocity = {
+            x: 0,
+            y: 0
+        }
+        this.width = 10
+        this.height = 10
+        this.collider = {
+            type: 'rectangle',
+            width: this.width,
+            height: this.height
+        };
+
+        this.lifetime = 3
```

Comment:
Nitpick for future code contributions to the game repos: any time you are creating a variable which represents some measurable unit, the variable name should include the unit, to make it maximally readable and understandable for somebody coming into the codebase fresh. in this case, this'd become `lifetimeInSeconds`